pyleoclim/tests/test_utils_tsutils.py

```diff
@@ -0,0 +1,170 @@
+
+import pytest
+from pyleoclim.utils import tsutils
+import numpy as np
+
+
+def test_time_unit_to_datum_exp_dir_unknown_time_unit():
+    # unknown time unit, gets default values
+    time_unit = 'unknown_value'
+    with pytest.warns(match='Time unit'):
+        (datum, exponent, direction) = tsutils.time_unit_to_datum_exp_dir(time_unit)
+        assert datum == 0
+        assert exponent == 0
+        assert direction == 'prograde'
+
+    with pytest.warns(match='Time unit'):
+        (datum, exponent, direction) = tsutils.time_unit_to_datum_exp_dir(time_unit, time_name='unknown')
+        assert direction == 'prograde'
+
+    with pytest.warns(match='Time unit'):
+        (datum, exponent, direction) = tsutils.time_unit_to_datum_exp_dir(time_unit, time_name='age')
+        assert direction == 'retrograde'   
```

Comment:
why not check `datum` and `exponent` too?

Reply:
Those are handled in the following tests (there is code following the `time_name` bit that modifies `datum` and `exponent`). This test set is trying to exclusively look at the `time_name` options and implications.

Reply:
@khider and I are confused about what this test does. Can you explain the goal?

Reply:
I think it's just to check what happens when some invalid `time_unit ` is passed

still though, I'd suggest adding
```
        assert datum == 0
        assert exponent == 0
```
, it's only 2 lines of code, makes it consistent with the code above, and the logic inside `time_unit_to_datum_exp_dir` may well change someday such that `time_name` also affects `datum` and `exponent`

Reply:
The goal of any test suite is to hit 100% coverage. That means we want the test suite to go down every pathway - every if/else, raise etc. 

So this test in particular checks to make sure we get the appropriate warning when 

1. time_unit is some unknown string, time_name is None
2. time_unit is some unknown string, time_name is some unknown string
3. time_unit is some unknown string, time_name is "age"